sdk/eventhub/azure-eventhubs/azure/eventhub/eventprocessor/__init__.py

```diff
@@ -13,5 +14,6 @@
     'EventProcessor',
     'PartitionProcessor',
     'PartitionManager',
+    'CheckpointManager',
```

Comment:
Is the user supposed to use the CheckpointManager directly? Creating new instances of it?

Reply:
No, user will use checkpoint_manager.update_checkpoint(), but they don't create a CheckpointManager